### PR TITLE
Showing a google map!

### PR DESCRIPTION
**Pull request by bmykietka**
OK so here it is, a working google maps view through ionic. For whatever reason I needed to run `ionic setup sass` (after installing gulp) in order to get the map to load properly and not show a white space. There may be a better way around this but I just wanted to get this up. See below for some inline comments.

### CODE REVIEW

File: ionic.project [hunk 1]
```diff
@@ -1,4 +1,12 @@
 {
   "name": "blindLunch",
-  "app_id": ""
+  "app_id": "",
```
Comment (bmykietka) [inline ionic.project:3]: Things changed by installing gulp?

File: www/index.html [hunk 2]
```diff
@@ -5,15 +5,13 @@
     <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
     <title></title>
 
-    <link href="lib/ionic/css/ionic.css" rel="stylesheet">
```
Comment (bmykietka) [inline www/index.html:8]: Not sure if we want this to happen or not, think this was from configuring sass?

File: www/index.html [hunk 3]
```diff
@@ -5,15 +5,13 @@
...
     <script src="lib/ionic/js/ionic.bundle.js"></script>
 
     <!-- cordova script (this will be a 404 during development) -->
     <script src="cordova.js"></script>
 
-    <!-- IF using Sass (run gulp sass first), then uncomment below and remove the CSS includes above
+    <!-- compiled css output -->
```
Comment (bmykietka) [inline www/index.html:13]: From sass configuration?

**Comment by TillaTheHun0** [comment]
Hmmm Sass is just preprocessing our ionic css, so perhaps some wonky css styling was preventing the map from showing. Thats my only guess.

**Comment by TillaTheHun0** [comment]
So I have a suggestion that we merge this now and open up an issue to track 'making the map work without sass'. Although I am not opposed to using sass for preprocessing. Just so other people can mess around with the map while authenticated.

**Comment by bmykietka** [comment]
@TillaTheHun0 Sounds good!

**Pull request merged**